src/components/KTreeList/KTreeDraggable.vue

```diff
@@ -152,7 +153,9 @@ const itemLabel = 'item-label'
 
 const iconSecondaryColor = (item: TreeListItem): string | undefined => {
   if (item.icon === 'documentList' || !item.icon) {
-    return item.selected ? 'var(--KTreeListItemSelectedBorder, var(--teal-200))' : 'var(--KTreeListItemUnselectedBorder, var(--grey-200))'
+    return item.selected
+      ? 'var(--KTreeListItemSelectedBorder, $tmp-color-teal-200)'
+      : 'var(--KTreeListItemUnselectedBorder, var(--kui-color-background-neutral-weaker, $kui-color-background-neutral-weaker))'
```

Comment:
issue: you have to use the JavaScript tokens here alongside the CSS custom properties -- Sass doesn't compile inside the script block